consts.go

```diff
@@ -22,9 +22,17 @@ const (
 	recoveryStateWipe
 )
 
-const (
-	metadataHeader  = "WAL"
-	metadataVersion = "1.0"
+var (
+	// "github.com/NebulousLabs/writeaheadlog\n"
+	metadataHeader = [38]byte{'g', 'i', 't', 'h', 'u', 'b', '.', 'c', 'o', 'm', '/',
```

Comment:
Do we need to have the full url as header?

Reply:
I think it's nice, gives you an easy way to figure out where the file came from if you aren't familiar with it already.